clusterloader2/pkg/measurement/util/phase_latency.go

```diff
@@ -102,7 +102,14 @@ func (o *ObjectTransitionTimes) CalculateTransitionsLatency(t map[string]Transit
 				klog.V(4).Infof("%s: failed to find %v time for %v", o.name, transition.To, key)
 				continue
 			}
-			lag = append(lag, latencyData{key: key, latency: toPhaseTime.Sub(fromPhaseTime)})
+			latencyTime := toPhaseTime.Sub(fromPhaseTime)
+			// latencyTime should be always larger than zero, however, in some cases, it might be a negative value
```

Comment:
Sorry - apparently I wasn't clear enough.
I didn't mean shortening the comment itself (though the current version is also fine).

I meant: making the lines shorter (i.e. splitting the lines earlier than you do now)

Reply:
should be fine now.